File: Server/src/data/requests/auth/AuthResponse.java
package data.requests.auth;

import data.UserAccount;

public class AuthResponse {
    public AuthResult Result;
    public UserAccount Data;
    public AuthResponse result(AuthResult result) {var a = this; a.Result = result; return a;}
    public AuthResponse data(UserAccount data) {var a = this; a.Data = data; return a;}
}

File: WinClient/Tester/AuthEntities/AuthResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;
using Web;

namespace Tester.AuthEntities
{
    public class AuthResponse
    {
        [JsonConverter(typeof(AuthResultConverter))]
        public AuthResult? Result { get; set; }
        public UserData? Data { get; set; }
        public string? PermCts { get; set; }
    }
}
